client/src/shared/filters/Capitalize.ts

```diff
@@ -0,0 +1,4 @@
+export default function (value:string) {
```

Comment:
What?.. 
Why not CSS text-transform: capitalize ?

Reply:
need to capitalize only first letter